data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -273,7 +273,7 @@
         <filename>debug_libvirtd.sh</filename>
         <source><![CDATA[
 cat >> /etc/libvirt/libvirtd.conf<< EOF
-log_level = 1
+log_level = 3
 log_filters="1:qemu 3:remote 4:event 3:util.json 3:rpc"
```

Comment:
Please have a look at https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15964#issuecomment-1340323599.

Reply:
BTW, host about host_12.xml.ep?

Reply:
Besides, the super large log (4G) is actually accumulated after long test sequence. If it is going to be avoided in most situations, I am afraid more needs to be done, given that we can't just set log_level to 3(simplified log, but not enough for dev to debug).
I remember our members used to share some useful ways, eg at 1/3 or 1/2 full test path of openqa modules in qem tests ( earlier modules should have passed, so current accumulated libvirt log is not needed  ), /var/log/libvirt/libvirtd.log can be removed and libvirtd service can be restarted to collect new logs for potential failure in following tests.
HDYT?

Reply:
> host about host_1

will think about sle12

Reply:
Add a new module to clear libvirtd.log before every test.